pages/how-to/set-up/new-mgmt-cluster.md

```diff
@@ -0,0 +1,104 @@
+---
+title: Setting Up a New MGMT Cluster
+description: Using Plural CLI to Deploy a MGMT Kubernetes Cluster
+---
+
+### Prerequisites
+[Plural CLI](/how-to/set-up/plural-cli)
+
+##### Ensure Cloud Provider CLI Authentication
+Plural uses the _default_ profile when deploying resources  
+
+AWS  
+```sh
+aws sts get-caller-identity
+```  
+AZ
+```sh
+az account show
+```
+GCP
+```sh
+gcloud auth list
+```
+
+
+### Create a New Repo for Plural
+```sh
+git clone git@github.com:pluralsh/plrl-how-to.git
+cd plrl-how-to
+```
+
+### Setup Repo and Deploy Resources
+Ensure your _[app.plural.sh](https://app.plural.sh/profile/me)_ User has `admin` permissions  
+Follow the onscreen prompts to setup the repo and deploy resources  
+* Use the provided Plural DNS Services for the MGMT Cluster
+* When providing a domain name provide the _canonical_ name, e.g. how-to-plrl.onplural.sh
+```sh
+plural login
+plural up
+```
+
+# Troubleshooting
+### Get Kubeconfig for the MGMT Cluster
+AWS 
+```sh
+aws eks update-kubeconfig --name [CLUSTER_NAME] 
```

Comment:
`plural wkspace kube-init` does that for you actually

Reply:
this is really cool!
I didn't know this feature existed